src/implementations/vanilla/components/social-media-follow/social-media-follow-print.scss

```diff
@@ -26,6 +28,8 @@ $social-media-follow-print: null !default;
 }
 
 .ecl-social-media-follow__list {
+  display: flex;
+  flex-direction: column;
```

Comment:
Why do we go vertical in print..? Was this requested, it takes much more space and being used at the bottom of the pages most of the times it doesn't seem a good idea to make it a vertical list, also because the alignment is pretty odd when we do it.

Reply:
I was wondering the same, and the best explanation I got was that we don't really have a choice, as the extra links would just take too much space horizontally (they can be quite long url)

Reply:
we could just use flex-wrap in print instead of going vertical, if that is the issue, but what can i say, let's keep doing things irrationally